Prosjektrapport-V2.md

```diff
@@ -81,10 +81,22 @@ Det er viktig å dokumentere ikke bare alle hensyn som tas, men også hvilke hen
 
 ### Aktører og UseCase(UC)
 
-### UC-diagram
+Definisjon: En UseCase er en beskrivelse av en sekvens av handlinger som utføres av brukerne innenfor er system, som gir observerbar verdi til en aktør. Aktøren kan være en person eller et annet system. UseCase fokuserer på hva systemet gjør og ikke hvordan det gjøres. 
+    
+Formål: Hovedformålet med UseCase er å fange opp funksjonelle krav ved å beskrive hvordan systemet skal oppføre seg i forskjellige sitasjoner. 
+    
+    
+
 
+### UC-diagram
+Her skal det settes in UC diagram
```

Comment:
Hvis det er et bilde, så bør det stå som en alt tekst/forklarende tekst for de som ikke får bildet opp eller ikke kan se bildet av forskjellige grunner, det er spesifikt bedt om av Ingrid.

Så for eksempel: Figur: UC-diagram